README.md

```diff
@@ -1 +1,216 @@
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Fetch Memes data**
+- **Render Memes**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://react-capstone-project.onrender.com/)
+
+-[Demonstration of the project](https://www.loom.com/share/0c730ec4404b4f5fbb12a296dd03ced2)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [ ] A PC running any operating system like Windows, Linux, or MacOS
+- [ ] An Internet connection
+- [ ] Knowledge of some basic Git commands
+- [ ] A browser like Brave, Google Chrome, Mozilla Firefox or Microsoft Edge
+- [ ] Git installed
+
+### Setup
+
+- [ ] Clone this repository to your desired folder:
+
+```
+git clone https://github.com/haftamudesta/React-Capstone-Project
+```
+
+```
+cd React-Capstone-Project
+```
+
+- [ ] Open the file in your code editor
+
+```
+code .
+```
+
+### Install
+
+- [ ] Node modules:
+
+```
+npm i
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+npm run dev
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+npm test
+```
+
+### Deployment
+
+You can build this project using:
+
+```
+npm  build
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **Haftamu Desta**
+
+- GitHub: [@haftamu](https://github.com/haftamudesta)
+- Twitter: [@DestaHaftamu](https://twitter.com/DestaHftamu?t=NQ4ovkdWbsfsjh62NFEXFg&s=09)
+- LinkedIn: [Haftamu Desta](https://www.linkedin.com/in/haftamu-desta-795791a1/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Improve design.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project then don't forget to give a star ⭐ on this repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank **![](https://img.shields.io/badge/Microverse-blueviolet).**
+I would like to thank **![](https://www.behance.net/sakwadesignstudio).**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
Please give proper credit to the author of the original design. I see that you have updated the link but did not change the `Microverse` badge text. This still does not qualify as giving proper credit to Nelson Sakwa since his name is not mentioned.

You can write your acknowledgment like this:

- I would like to thank [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) for the project inspiration, as required by the [Creative Commons License](https://creativecommons.org/licenses/)